recipes/pyscenic/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "pyscenic" %}
+{% set version = "0.12.1" %}
+{% set sha256 = "ae8fafa707d2578ffe08f9eed85f14a4cd9e1b53d57217420e2e956f0a8ddba2" %}
+{% set patch_sha256 = "9e31c0fa80bf94481856d6670ad503ae1f40e32a654d03ed0fc757fba1bfee15" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: "{{ sha256 }}"
+  - path: dill_patch.patch
+    sha256: "{{ patch_sha256 }}"

```

Comment:
It seems to be that in other recipes they handle patches like this:

```
source:
  url: https://github.com/kcleal/gw/archive/refs/tags/v{{ version }}.tar.gz
  sha256: {{ sha256 }}
  patches:
    - fix-linker-issues.patch  # [linux]
    - fix-context.patch
```

and they don't seem to do anything on the build.sh. Can you please try like this and removing the patching part from the build (I suspect you might not need the build if this patching mechanism works).